### ISSUE TITLE
Fix Request: false positive in AvoidTimeUnitConfusion

### ISSUE THREAD

**Issue by jborgers** [jborgers/PMD-jPinpoint-rules#630]
When type is Duration, it knows about the time unit, so no violation.
```
class SomeListener {
   public SomeListener(@Value("${expire-duration}") Duration expireDuration) { // <-- false positive
      this.expireDuration = expireDuration;
   }
}
```
